### ISSUE TITLE
Input image should be normalized as follows:

### ISSUE THREAD

**Issue by chenbolinstudent** [Res2Net/Res2Net-PretrainedModels#18]
Input image should be normalized as follows:

normalize = transforms.Normalize(mean=[0.485, 0.456, 0.406],
                                  std=[0.229, 0.224, 0.225])

can you tell me how to do it in detail and give a example?
thanks

**Comment by chenbolinstudent** [comment]
and if i do not normalized, how about the result ?

**Comment by gasvn** [comment]
You can refer to the example code of training ImageNet of pytorch to see how to implement normalization.
https://github.com/pytorch/examples/blob/master/imagenet/main.py

**Comment by gasvn** [comment]
> and if i do not normalized, how about the result ?

If the data distribution is much different from the data used for training the pretrained model, the parameters in pretrained model will have limited help when applied to new data.
